index.html

```diff
@@ -278,6 +278,69 @@ <h3>
           reset because its [=permission/lifetime=] has expired.
         </p>
       </section>
+      <section>
+        <h3>
+          Permission Store
+        </h3>
+        <p>
+          The user agent maintains a single <dfn class="export">permission store</dfn> which is a [=/list=] of [=permission store entries=].
+        </p>
+        <p>
+          The user agent MAY remove [=entries=] from the [=permission store=] when their respective [=permission=]'s [=permission/lifetime=] has expired.
+        </p>
+        <p>
+          The user agent MAY maintain additional permission stores with [=implementation-defined=] scope and eviction rules. Which permission store is adressed in a given moment is [=implementation-defined=].
+          <aside class="note">
+            This is intended to allow a user agent to experiment with user-friendly permission concepts such as per-tab grants.
+          </aside>
+        </p>
+        <p>
+          A <dfn class="export" data-local-lt="entry">permission store entry</dfn> is a [=tuple=] of [=powerful feature/name=] <dfn>name</dfn>, [=permission store key=] <dfn>key</dfn>, {{PermissionDescriptor}} <dfn>descriptor</dfn>, and [=permission/state=] <dfn>state</dfn>.
```

Comment:
I wonder if we can avoid `<dfn>name</dfn>` and `<dfn>key</dfn>` if they are just aliases of the other concepts? We could probably just `data-local-lt=` them instead.

Reply:
No, they're members of the entry. They should probably have `for="permission store entry"` though.